editoast/src/views/paced_train.rs

```diff
@@ -402,27 +403,62 @@ async fn simulation(
 )]
 async fn project_path(
```

Comment:
Can we improve the documentation of this endpoint to make explicit that we do project only one train schedule per paced train, and that train schedule is the first occurrence of the paced train. It's not necessary trivial in this API what is being projected.